source/_templates/installations/common/firewall-ports-note.rst

```diff
@@ -0,0 +1,3 @@
+.. note::
+   
+   CentOS systems have a firewall enabled by default that may block communication between Wazuh components if they are spread across different hosts. Ensure that the necessary ports are open in your firewall configuration. For a complete list of required ports, refer to the :ref:`Required ports <default_ports>` section in the architecture documentation.
```

Comment:
```suggestion
Note that firewalls can block communication between Wazuh components on different hosts. Refer to the :ref:`Required ports <default_ports>` section and ensure the necessary ports are open.
```

Reply:
I would add as few notes as possible. I think adding it as a paragraph can work well.